data/test/scenarios/test_force_chance_to_hit_macro.cfg

```diff
@@ -37,6 +37,15 @@
                     [/chance_to_hit]

```

Comment:
This `[object]` now has two different `[chance_to_hit]` blocks. IIUC, a comment saying "check that the macro overrides both abilities and weapon-specials" would help.

The old documentation at the top of the file talks about the `unupgradable` trick, but that doesn't seem to match anything in this test.